cstar/tests/unit_tests/system/test_environment.py

```diff
@@ -440,9 +400,8 @@ def test_env_file_load_count(
 
             env.set_env_var(new_var, new_value)
 
-            # update should trigger load from disk
-            assert loader.call_count == initial_num_calls + 1
-            assert new_var in env.environment_variables
+            # update should not trigger new load from disk
+            assert loader.call_count == initial_num_calls
```

Comment:
This assertion will pass, but it makes no sense and could just be omitted (dotenv is not reloading the environment by writing the .cstar.env file). This probably means the entire test can go